omero/developers/Tables.rst

```diff
@@ -9,7 +9,7 @@ Large and small volumes of tabular data can be stored via named columns, and
 retrieved in bulk or via paging. A limited query language provides basic
 filtering and selecting.
 
-Since 5.6, `PyTables <http://pytables.org>`__ are installed when installing
+Since 5.6, `PyTables`_ are installed when installing
```

Comment:
I don't see PyTables as a dependency of omero-py. I had to install pytables separately.

Reply:
This means that our installation guides do not have any more information about pytables installation.
Since it was removed in omero-install
https://github.com/ome/omero-install/pull/216/files

Reply:
Indeed, also on my machine the pip install omero-py does not seem to have installed tables

```
(base) pwalczysko@ls31618~$ conda activate myenv
(myenv) pwalczysko@ls31618~$ pip freeze
certifi==2019.11.28
future==0.18.2
numpy==1.18.1
omero-py==5.6.0
Pillow==7.0.0
zeroc-ice==3.6.5
```